CMakeLists.txt

```diff
@@ -124,6 +138,7 @@ endif()
 
 if (BUILD_CXX_LANGUAGE_PACKAGE)
 
+	find_package(LLVM REQUIRED)
```

Comment:
This should not be necessary. We used to have it, but removed it when revising this CMakeLists for open-sourcing. It should work when the environment variable `Clang_DIR` is set to the main `build` directory of LLVM (I think it is wrong in the README where it points to a subdirectory in the build directory).